server/routes/dashboardActivity.tsx

```diff
@@ -41,6 +41,7 @@ app.get(
 						title: `Activity · ${initialData.scopeData.elements.activeTarget.title}`,
 						unlisted: true,
 					})}
+					bodyClassPrefix="dashboard"
```

Comment:
Ditto here — the dashboard won't ever see custom CSS so we might as well omit the prop in that case.